es/refresher-linear-algebra.md

```diff
@@ -228,44 +228,43 @@
 
-&#10230; Dependencia Lineal ― Un conjunto de vectores se dice que es linealmente dependiente si uno de los vectores en el grupo puede ser definido como una combinación lineal de los otros.
+&#10230; Dependencia lineal ― Un conjunto de vectores se dice que es linealmente dependiente si uno de los vectores del conjunto puede ser definido como una combinación lineal de los restantes.
 
 <br>
 
 **40. Remark: if no vector can be written this way, then the vectors are said to be linearly independent**
 
-&#10230; Nota: si no se puede escribir el vector de esta manera, entonces el vector se dice que es linealmente independiente
+&#10230; Observación: si ningún vector puede ser escrito de esta forma, entonces se dice que los vectores son linealmente independientes
 
 <br>
 
 **41. Matrix rank ― The rank of a given matrix A is noted rank(A) and is the dimension of the vector space generated by its columns. This is equivalent to the maximum number of linearly independent columns of A.**
 
-&#10230; Rango matricial ― El rango de una matriz A, nombrado rank(A), que es la dimensión del vector espacial generado por sus columnas. Lo que es equivalente al número máximo de columnas linealmente independientes de A.
+&#10230; Rango matricial ― El rango de una matriz A, denotado rank(A), es la dimensión del espacio vectorial generado por sus columnas. Esto es equivalente al máximo número de columnas linealmente independientes de A.
  
 <br>
 
 **42. Positive semi-definite matrix ― A matrix A∈Rn×n is positive semi-definite (PSD) and is noted A⪰0 if we have:**
 
-&#10230; Matriz semi-definida positiva ― Una matriz A∈Rn×n es semi-defininda positivamente (PSD) y se tiene que A⪰0 si:
+&#10230; Matriz semi-definida positiva ― Una matriz A∈Rn×n es semi-defininda positiva (PSD), lo cual se denota como A⪰0, si se cumple que:
 
 <br>
 
 **43. Remark: similarly, a matrix A is said to be positive definite, and is noted A≻0, if it is a PSD matrix which satisfies for all non-zero vector x, xTAx>0.**
 
-&#10230; Nota: de igual forma, una matriz A se dice positiva y definida, A≻0, si esa una matriz PSD que satisface para todos los vectores diferentes de cero x, xTAx>0.
+&#10230; Observación: de igual forma, una matriz A se dice definida positiva, lo cual se denota con A≻0, si es una matriz PSD que satisface para todos los vectores x diferentes de cero, xTAx>0.
 
 <br>
 
 **44. Eigenvalue, eigenvector ― Given a matrix A∈Rn×n, λ is said to be an eigenvalue of A if there exists a vector z∈Rn∖{0}, called eigenvector, such that we have:**
 
-&#10230; Eigenvalor, eigenvector ― Dado una matriz A∈Rn×n, λ se dice que es el eigenvalor de A si existe un vector z∈Rn∖{0}, llamado eigenvector, tal que se tiene:
+&#10230; Valor propio, vector propio ― Dada una matriz A∈Rn×n, λ se dice que es un valor propio de A si existe un vector z∈Rn∖{0}, llamado vector propio, tal que:
```

Comment:
- I'm not sure if "(Eigenvalue, eigenvector en Inglés)" should be included for these terms.

Reply:
I also agree with gustavo. Since it's very common to find these terms in Spanish literature, let's add them like "_(eigenvalue, en inglés)_" or "_(eigenvector, en inglés)_".

Reply:
I wrote it in the description. For me it's clear enough. Do you agree with it or prefer adding it in the title?